src/plot_api/plot_api.js

```diff
@@ -750,6 +751,17 @@ function cleanData(data, existingData) {
             }
         }
 
+        // fix typo in colorscale definition
```

Comment:
`cleanData` is pretty up-front about its [purpose](https://github.com/plotly/plotly.js/blob/Yl-not-YI/src/plot_api/plot_api.js#L689).